### PR TITLE
Update styling of graph settings textbox and add shadow

### LINKED ISSUE #986: Add shadow elements to graphing calculator wherever we have custom popups
**Describe the bug**
Some custom popups do not have shadows.

**Expected behavior**
Audit all custom popups added for graphing mode and ensure shadows are enabled for consistency with platform.

**Screenshots**
image

### LINKED ISSUE #1018: Graph window options x/y min/max textbox styling does not match variable min/max styling
**Describe the bug**
Graph window options x/y min/max textbox styling does not match variable min/max styling.

**Expected behavior**
Styling should be consistent with variable editing. As we add more graph options, we will also benefit from the reduced visual footprint of this styling.
image

**Screenshots**
image

### PR DESCRIPTION
**Pull request by joseartrivera**
### Description of the changes:
- Updates text boxes in settings flyout to be consistent with variable text boxes
- Adds shadow to trace value pop up

### How changes were validated:
- Manual tests

### CODE REVIEW

File: src/Calculator/App.xaml [hunk 1]
```diff
@@ -85,6 +85,22 @@
                     <SolidColorBrush x:Key="EquationBrush13" Color="#FF8E562E"/>
                     <SolidColorBrush x:Key="EquationBrush14" Color="#FF000000"/>
 
+                    <SolidColorBrush x:Key="GraphSettingsErrorBackgroundBrush" Color="#33EB5757"/>
+                    <SolidColorBrush x:Key="GraphSettingsErrorBorderBrush" Color="#FFEB5757"/>
+                    <SolidColorBrush x:Key="TextControlBackgroundFocused" Color="Transparent"/>
```
Comment (rudyhuyn) [inline src/Calculator/App.xaml]: These brushes override system brushes and will modify all textboxes in the app. Their scopes were limited when they were in `EquationInputArea` but not here.

File: src/Calculator/App.xaml [hunk 2]
```diff
@@ -197,6 +228,29 @@
                     <SolidColorBrush x:Key="EquationBrush2" Color="{ThemeResource SystemColorHighlightColor}"/>
                     <SolidColorBrush x:Key="EquationBrush3" Color="{ThemeResource SystemColorHotlightColor}"/>
                     <SolidColorBrush x:Key="EquationBrush4" Color="{ThemeResource SystemColorWindowTextColor}"/>
+
+                    <SolidColorBrush x:Key="GraphSettingsErrorBackgroundBrush" Color="{ThemeResource SystemColorButtonFaceColor}"/>
+                    <SolidColorBrush x:Key="GraphSettingsErrorBorderBrush" Color="Red"/>
+                    <SolidColorBrush x:Key="TextControlBackgroundFocused" Color="{StaticResource SystemColorButtonFaceColor}"/>
+                    <SolidColorBrush x:Key="TextControlBackgroundPointerOver" Color="{StaticResource SystemColorButtonFaceColor}"/>
+                    <SolidColorBrush x:Key="TextControlForegroundFocused" Color="{StaticResource SystemColorButtonTextColor}"/>
+                    <SolidColorBrush x:Key="VariablesBackgroundBrush" Color="{StaticResource SystemColorBackgroundColor}"/>
```
Comment (rudyhuyn) [inline src/Calculator/App.xaml]: Not related to your PR, but this brush is not used, let's remove it

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <UserControl x:Class="CalculatorApp.GraphingCalculator"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:Windows10version1903="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 8)"
```
Comment (rudyhuyn) [inline src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: We should not associate UniversalApiContract and windows versions and be consistent with other namespaces.

`xmlns:contract8Present="..."`

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp [hunk 4]
```diff
@@ -87,6 +87,8 @@ GraphingCalculator::GraphingCalculator()
     virtualKey->Key = (VirtualKey)187; // OemAdd key
     virtualKey->Modifiers = VirtualKeyModifiers::Control;
     ZoomInButton->KeyboardAccelerators->Append(virtualKey);
+
+    SharedShadow->Receivers->Append(GraphingControl);
```
Comment (rudyhuyn) [inline src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp]: This code will crash on older version of windows

File: src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cpp [hunk 5]
```diff
@@ -37,11 +37,11 @@ Style ^ GraphingSettings::SelectTextBoxStyle(bool incorrectRange, bool error)
 {
     if (incorrectRange || error)
     {
-        return static_cast<::Style ^>(this->Resources->Lookup(L"ErrorTextBoxStyle"));
+        return static_cast<::Style ^>(Application::Current->Resources->Lookup(L"VariableTextErrorBoxStyle"));
     }
     else
     {
-        return nullptr;
+        return static_cast<::Style ^>(Application::Current->Resources->Lookup(L"ThemedVariableTextBoxStyle"));
```
Comment (rudyhuyn) [inline src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cpp]: The textbox won't be updated when the user will modify the app theme. 

We should try something like:
`<Style x:Key="GraphSettingsVariableTextBoxStyle" BasedOn="{ThemeResource ThemedVariableTextBoxStyle}" />` (in GraphingSettings.xaml)
and write
`return static_cast<::Style ^>(this->Resources->Lookup(L"GraphSettingsVariableTextBoxStyle"));`

File: src/Calculator/App.xaml (hunk 1 above)
Comment (joseartrivera) [reply src/Calculator/App.xaml]: Good catch!

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp (hunk 4 above)
Comment (joseartrivera) [reply src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp]: This code shouldn’t run on older versions since it’s Graphing Calc but it’s probably good to add an explicit check anyway.

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml (hunk 3 above)
Comment (joseartrivera) [reply src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: Is there a reason we shouldn’t or is it just for consistency? It is what VS auto-inserted for me, should we send feedback for them to change that?

File: src/Calculator/App.xaml [hunk 6]
```diff
@@ -85,6 +85,22 @@
                     <SolidColorBrush x:Key="EquationBrush13" Color="#FF8E562E"/>
                     <SolidColorBrush x:Key="EquationBrush14" Color="#FF000000"/>
 
+                    <SolidColorBrush x:Key="GraphSettingsErrorBackgroundBrush" Color="#33EB5757"/>
```
[the remark this answers is not in the document]
Comment (joseartrivera) [reply src/Calculator/App.xaml]: They are already removed!

File: src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cpp (hunk 5 above)
Comment (joseartrivera) [reply src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cpp]: Hmm, are you sure? It seems to be working for me

Comment (rudyhuyn) [reply src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cpp]: it works because GraphSettings is automatically closed when the window loses the focus. But if we want this popup to persist or when the theme is changed automatically (for example one of the many apps switching your theme automatically at night), the style used here won't be updated.

Comment (joseartrivera) [reply src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml.cpp]: I made the change, but I couldn't use ThemeResource because I got this error:
Style BasedOn property must be a Style, not 'ThemeResourceExtension' object

Still works for me, but unsure if the automatic theme changing will function correctly.

File: src/Calculator/App.xaml [hunk 7]
```diff
@@ -1766,6 +1810,29 @@
...
+                <Setter Property="Padding" Value="2,6,2,2"/>
+                <Setter Property="TextAlignment" Value="Center"/>
+                <Setter Property="Background" Value="Transparent"/>
+                <Setter Property="BorderThickness" Value="0,0,0,1"/>
+                <Setter Property="FontSize" Value="14"/>
+                <Setter Property="InputScope" Value="Number"/>
+                <Setter Property="MinWidth" Value="30"/>
+            </Style>
+
+            <Style x:Key="VariableTextErrorBoxStyle" TargetType="TextBox">
```
Comment (rudyhuyn) [inline src/Calculator/App.xaml:1825]: Except the background and the borderbrush, this style is exactly the same than VariableTextBoxStyle.

What do you think about replacing it with:
```
<Style x:Key="VariableTextErrorBoxStyle"
                           BasedOn="{StaticResource VariableTextBoxStyle}"
                           TargetType="TextBox">
    <Setter Property="BorderBrush" Value="{ThemeResource GraphSettingsErrorBorderBrush}"/>
    <Setter Property="Background" Value="{ThemeResource GraphSettingsErrorBackgroundBrush}"/>
</Style>
```
?

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp [hunk 8]
```diff
@@ -619,7 +624,7 @@ void CalculatorApp::GraphingCalculator::AddTracePointerShadow()
     dropShadow->Mask = CursorPath->GetAlphaMask();
 
     auto shadowSpriteVisual = compositor->CreateSpriteVisual();
-    shadowSpriteVisual->Size = ::Numerics::float2(static_cast<float>(CursorPath->ActualWidth), static_cast<float>(CursorPath->ActualHeight));
+    shadowSpriteVisual->Size = ::Numerics::float2(18, 18);
```
Comment (rudyhuyn) [inline src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp:627]: Why this change?

File: src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml [hunk 9]
```diff
@@ -223,8 +226,7 @@
                        AutomationProperties.HeadingLevel="Level2"/>
 
             <StackPanel Orientation="Vertical">
-                <RadioButton x:Uid="AlwaysLightTheme"
-                             IsChecked="{x:Bind IsMatchAppTheme, Mode=TwoWay, Converter={StaticResource BooleanNegationConverter}}"/>
+                <RadioButton x:Uid="AlwaysLightTheme" IsChecked="{x:Bind IsMatchAppTheme, Mode=TwoWay, Converter={StaticResource BooleanNegationConverter}}"/>
```
Comment (rudyhuyn) [inline src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml:229]: Can we use more explicit name? `ThemeOptionAlwaysLight`/ `ThemeOptionMatchApp`

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp (hunk 8 above)
Comment (joseartrivera) [reply src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp:627]: ActualWidth and ActualHeight are usually 0 at the time this code runs

File: src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml (hunk 9 above)
Comment (joseartrivera) [reply src/Calculator/Views/GraphingCalculator/GraphingSettings.xaml:229]: Hmm agree, but not part of this PR, this is just XAML styler rearranging things
